backend/app/models/__init__.py

```diff
@@ -9,6 +9,7 @@ def init_app(app):
     from .entity import Entity
     from .user import User
     from .sign_in_logs import SignInLogs
+    from .residences import Residence
```

Comment:
I think these changes should be renamed. Instead of `residence/residences` it should be `resident/residents` everywhere in the code base (since we are dealing with residents instead of residence buildings`

Reply:
big oops, my english is so bad 🥸😩💀👨🏻‍🦲